lib/matplotlib/legend.py

```diff
@@ -257,6 +257,10 @@ def _update_bbox_to_anchor(self, loc_in_canvas):
     to set the fontsize alongside other font properties, use the *size*
     parameter in *title_fontproperties*.
 
+alignment : {'center', 'left', 'right'}, default: 'center'
+    Control the alignment of the legend box (title and handles).
+    Elements will align to the longest one.
```

Comment:
```suggestion
    The alignment of the legend title and the box of entries. The entries
    are aligned as a single block, so that markers always lined up.
```
I think this is what we need to communicate. "Align to the longest one"
is technically correct, but harder to understand.